libs/types/src/tokens.rs

```diff
@@ -395,44 +395,6 @@ impl CrossChainTransferability {
 	}
 }
 
-/// Liquidity Pools-wrapped tokens
-///
-/// Currently, LiquidityPools are only deployed on EVM-based chains and
-/// therefore we only support EVM tokens. In the far future, we might support
-/// wrapped tokens from other chains such as Cosmos based ones.
-#[derive(
-	Clone,
-	Copy,
-	PartialOrd,
-	Ord,
-	PartialEq,
-	Eq,
-	Debug,
-	Encode,
-	Decode,
-	TypeInfo,
-	MaxEncodedLen,
-	Serialize,
-	Deserialize,
-)]
-pub enum LiquidityPoolsWrappedToken {
```

Comment:
It seems it can be removed. 

Its only current purpose is to return the inner values in the private method of `pallet-liquidity-pools`: `try_get_wrapped_token()`. From my PoV, it adds a new layer of understanding for the reading; doing that as a tuple makes things easier. 

I can revert if you have some point against it.